plugins/obs-webrtc/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+project(obs-webrtc)
+
+include(FetchContent)
+
+FetchContent_Declare(
+  Corrosion
+  GIT_REPOSITORY https://github.com/corrosion-rs/corrosion.git
+  GIT_TAG v0.3.1)
+
+FetchContent_MakeAvailable(Corrosion)
+
+set(obs-webrtc_HEADERS whip-output.h)
+set(obs-webrtc_SOURCES whip-service.c whip-output.c webrtc.c)
+
+if(WIN32)
+  set(MODULE_DESCRIPTION "OBS webrtc module")
+  configure_file(${CMAKE_SOURCE_DIR}/cmake/bundle/windows/obs-module.rc.in
```

Comment:
Always e close directory/file specifications that are composed with user-defined variables in double quotes.